server.js

```diff
@@ -131,6 +132,49 @@ server.get('/api/workers', function (req, res) {
 
 });
 
+server.get('/api/invoices', async function (req, res) {
+    const {moneybird} = yaml.load(await fetch.fetchFromFile(req, 'subscription.yml'))
+
+    console.log('Fetching invoice data from moneybird: ', moneybird)
+    const invoices = await getInvoices(moneybird.contact_id, moneybird.reference);
+
+    res.status(200).send(invoices);
+});
+
+server.get('/api/invoice_contact', async function (req, res) {
+
+    const {moneybird} = yaml.load(await fetch.fetchFromFile(req, 'subscription.yml'))
+    console.log('Fetching contact data from moneybird')
+    const contact_data = await getContactData(moneybird.contact_id);
+
+    res.status(200).send(contact_data);
+});
+
+server.get('/api/invoices', function (req, res) {
```

Comment:
What does this do? `server.get('/api/invoices')` is also implemented on line 135.

Reply:
nothing. I'll remove it